openmls/src/treesync/treekem.rs

```diff
@@ -211,36 +209,46 @@ impl PlaintextSecret {
     /// Prepare the `GroupSecrets` for a number of `invited_members` based on a
     /// [`TreeSyncDiff`]. If a slice of [`PlainUpdatePathNode`] is given, they
     /// are included in the [`GroupSecrets`] of the path.
+    ///
+    /// Returns an error if
```

Comment:
Shouldn't the points on the list below be assumptions on the inputs rather than errors? In other cases, we don't document the cases where functions throw library errors.

Reply:
I'm not sure I understand. This is the list of assumptions. We have this disclaimer on many (most?) functions now.

Reply:
Never mind. I thought it makes sense here between assumptions (breaking which results in a `LibraryError` and actual error cases, but since we don't really have those any more, it doesn't make much of a difference.